src/program.js

```diff
@@ -167,6 +167,11 @@ Example: $0 --help run.
         demand: true,
         type: 'string',
       },
+      'timeout' : {
+        describe: 'A length in millesconds to give up after if the add-on ' +
+                  'hasn\'t been signed',
```

Comment:
> I think this message can be more concise

I copied the message from the https://github.com/mozilla/sign-addon source and added in another word. I will go ahead and make the modification if you think that is better.